pkg/controller/statusmanager/status_manager.go

```diff
@@ -58,6 +58,22 @@ func (status *StatusManager) Set(reachedAvailableLevel bool, conditions ...confi
 
 	oldStatus := co.Status.DeepCopy()
 
+	related := []configv1.ObjectReference{
+		{
+			Resource: "namespaces",
+			Name:     "openshift-sdn",
```

Comment:
(Aside: is "namespace**s**" correct? I would have expected it to want the singular.)

This should be computed dynamically based on the actual Namespaces we created. The same way that ReconcileOperConfig figures out the set of active DaemonSets and Deployments and then tells the StatusManager about them, it should do the same with Namespaces. (Except it should add the controller's namespace to the list too.)

Reply:
Plural seems fine, I found it in other spots of the code:

https://github.com/openshift/machine-api-operator/blob/6066765d8ff090fd081c7a20e59955d00004b0ad/pkg/operator/status.go#L165
https://github.com/openshift/cluster-samples-operator/blob/df3dbf4d8a119058f80067eca568c6f78ff8d725/pkg/operatorstatus/operatorstatus.go#L169
https://github.com/openshift/cluster-autoscaler-operator/blob/62768a6ba48010d12c4d86b8b06689a6970a7841/pkg/operator/operator.go#L110

Reply:
Plural is correct, because this is a Resource, not a Kind. However, dan is right, this should be computed automatically.